daprdocs/content/en/reference/components-reference/supported-pubsub/_index.md

```diff
@@ -48,5 +48,5 @@ Table captions:
 
 | Name                                                      | Status | Component version | Since |
 |-----------------------------------------------------------|--------| ----------------| -- |
-| [Azure Event Hubs]({{< ref setup-azure-eventhubs.md >}})  | Alpha  | v1 | 1.0 |
+| [Azure Event Hubs]({{< ref setup-azure-eventhubs.md >}})  | Beta  | v1 | 1.6 |
```

Comment:
Add link to the components-contrib PR to this this PR for tracking

Reply:
done